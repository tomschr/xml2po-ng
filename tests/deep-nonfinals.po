msgid ""
msgstr ""
"Project-Id-Version: друга проба\n"
"POT-Creation-Date: 2005-03-27 13:44+0200\n"
"PO-Revision-Date: 2005-03-27 13:44+0200\n"
"Last-Translator: Данило <EMAIL@ADDRESS>\n"
"Language-Team: српски <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#. When image changes, this message will be marked fuzzy or untranslated for you.
#. It doesn't matter what you translate it to: it's not used at all.
#: deep-nonfinals.xml:18 (None)
msgid "@@image: 'figure-applet-modem.png'; md5=THIS FILE DOESN'T EXIST"
msgstr "@@image: 'figure-applet-modem.png'; md5=THIS FILE DOESN'T EXIST"

#. When image changes, this message will be marked fuzzy or untranslated for you.
#. It doesn't matter what you translate it to: it's not used at all.
#: deep-nonfinals.xml:40 (None)
msgid "@@image: 'figure-applet-trash.png'; md5=THIS FILE DOESN'T EXIST"
msgstr "@@image: 'figure-applet-trash.png'; md5=THIS FILE DOESN'T EXIST"

#: deep-nonfinals.xml:7 (title)
msgid "Panel Items"
msgstr "Ставке панела"

#: deep-nonfinals.xml:9 (title)
msgid "Integrated Modem Control"
msgstr "Интегрисано управљање модемом"

#: deep-nonfinals.xml:10 (para)
msgid ""
"GNOME 2.10 introduces a new applet for controlling your Modem, integrated "
"with GNOME System Tools."
msgstr ""
"Гном 2.10 уводи ново програмче за управљање вашим модемом, интегрисано са "
"Системским алатима Гнома."

#: deep-nonfinals.xml:15 (title) deep-nonfinals.xml:20 (phrase)
msgid "The new modem control applet"
msgstr "Ново управљачко програмче модема"

#: deep-nonfinals.xml:28 (title)
msgid "Panel Trashcan"
msgstr "Смеће на панелу"

#: deep-nonfinals.xml:30 (para)
msgid ""
"GNOME 2.10 features an optional trashcan for your Panel. Just like the "
"trashcan on your desktop, you can now drag and drop files and other items "
"into the trash without minimizing your open windows."
msgstr ""
"Гном 2.10 доноси необавезну канту за смеће на ваш панел. Као и смеће на "
"радној површи, можете превући и пустити датотеке и друге ставке у смеће без "
"умањивања ваших отворених прозора."

#: deep-nonfinals.xml:37 (title)
msgid "A trashcan on your Panel"
msgstr "Канта за смеће на вашем панелу"

#: deep-nonfinals.xml:42 (phrase)
msgid "Drag trash to your panel"
msgstr "Одвуците смеће на ваш панел"

#. Put one translator per line, in the form of NAME <EMAIL>, YEAR1, YEAR2
#: deep-nonfinals.xml:0 (None)
msgid "translator-credits"
msgstr "Данило Шеган"
