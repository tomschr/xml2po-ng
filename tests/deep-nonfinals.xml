<?xml version="1.0"?>
<!DOCTYPE sect2 PUBLIC "-//OASIS//DTD DocBook XML V4.1.2//EN"
                  "http://www.oasis-open.org/docbook/xml/4.1.2/docbookx.dtd" [
<!ENTITY version "2.10">
]>
<sect2 id="rnapplets">
  <title>Panel Items</title>
  <sect3 id="rnapplets-modemlights">
  <title>Integrated Modem Control</title>
  <para>
   GNOME &version; introduces a new applet for controlling your Modem, integrated with GNOME System Tools.
  </para>

   <figure id="rnscreenshot-applet-modem">
    <title>The new modem control applet</title>
      <screenshot>
        <mediaobject>
          <imageobject><imagedata fileref="figure-applet-modem.png" format="PNG"/>
          </imageobject>
        <textobject><phrase>The new modem control applet</phrase></textobject>
      </mediaobject>
    </screenshot>
  </figure>

  </sect3>

  <sect3 id="rnapplets-trash">
  <title>Panel Trashcan</title>

  <para>
   GNOME &version; features an optional trashcan for your Panel.
   Just like the trashcan on your desktop, you can now drag and drop files and other items into the trash
   without minimizing your open windows.
  </para>

  <figure id="rnscreenshot-applet-trash">
    <title>A trashcan on your Panel</title>
      <screenshot>
        <mediaobject>
          <imageobject><imagedata fileref="figure-applet-trash.png" format="PNG"/>
          </imageobject>
        <textobject><phrase>Drag trash to your panel</phrase></textobject>
      </mediaobject>
    </screenshot>
  </figure>
  </sect3>
</sect2>
