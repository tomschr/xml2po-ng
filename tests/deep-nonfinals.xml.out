<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE sect2 PUBLIC "-//OASIS//DTD DocBook XML V4.1.2//EN" "http://www.oasis-open.org/docbook/xml/4.1.2/docbookx.dtd" [
<!ENTITY version "2.10">
]>
<sect2 id="rnapplets">
  <title>Ставке панела</title>
  <sect3 id="rnapplets-modemlights">
  <title>Интегрисано управљање модемом</title>
  <para>Гном 2.10 уводи ново програмче за управљање вашим модемом, интегрисано са Системским алатима Гнома.</para>

   <figure id="rnscreenshot-applet-modem">
    <title>Ново управљачко програмче модема</title>
      <screenshot>
        <mediaobject>
          <imageobject><imagedata fileref="figure-applet-modem.png" format="PNG"/>
          </imageobject>
        <textobject><phrase>Ново управљачко програмче модема</phrase></textobject>
      </mediaobject>
    </screenshot>
  </figure>

  </sect3>

  <sect3 id="rnapplets-trash">
  <title>Смеће на панелу</title>

  <para>Гном 2.10 доноси необавезну канту за смеће на ваш панел. Као и смеће на радној површи, можете превући и пустити датотеке и друге ставке у смеће без умањивања ваших отворених прозора.</para>

  <figure id="rnscreenshot-applet-trash">
    <title>Канта за смеће на вашем панелу</title>
      <screenshot>
        <mediaobject>
          <imageobject><imagedata fileref="figure-applet-trash.png" format="PNG"/>
          </imageobject>
        <textobject><phrase>Одвуците смеће на ваш панел</phrase></textobject>
      </mediaobject>
    </screenshot>
  </figure>
  </sect3>
</sect2>
